### ISSUE TITLE
Lottie streaming

### ISSUE THREAD

**Issue by zwjohn** [airbnb/lottie-web#2651]
I was wondering if there is a feature to stream animation data or a plan to implement a such feature in the future. Currently you have to load different data json to start animation for the same object, it would be nice to be able to stream the animation data, so that the animation can be continuous without reloading the object.

Thanks!

**Comment by bodymovin** [comment]
hi, do you mean adding dynamically newer animations to the original json?

**Comment by zwjohn** [comment]
@bodymovin Yes, something like that, so that the animation data can be continuously streamed from a server to UI. The streamed data are part of an existing animation, the player renders the new animation data dynamically without reloading.

**Comment by AkromDev** [comment]
@zwjohn Did you find any solution?

**Comment by zwjohn** [comment]
Nope, you still need to load the whole JSON to play on the browser. The only improvement is that there is now a compressed version of the JSON which makes the file smaller, but it's a paid service.
